apps/emqx_bridge_es/src/emqx_bridge_es.erl

```diff
@@ -30,22 +30,14 @@ fields(action) ->
             }
         )};
 fields(action_config) ->
-    emqx_resource_schema:override(
-        emqx_bridge_v2_schema:make_consumer_action_schema(
```

Comment:
This is an action (producer), not a source (consumer).  Hence the new common field was not picked up automatically.